engine/classes/Elgg/Notifications/SubscriptionsService.php

```diff
@@ -617,6 +617,10 @@ protected function getSubscriptionRecords(array $container_guid, array $methods,
 		if (!$rels) {
 			return [];
 		}
+		
+		if (empty(array_unique(array_filter($container_guid)))) {
```

Comment:
why not do `$container_guid = array_unique(array_filter($container_guid));` fo futher use in the function